.travis.yml

```diff
@@ -50,7 +50,7 @@ notifications:
 
 install:
   - pip3 install -U -q pip
-  - pip3 install -U -q scikit-optimize hyperopt hpbandster ConfigSpace scipy dataclasses optuna
+  - pip3 install -U -q scikit-optimize==0.8.1 hyperopt==0.2.5 hpbandster==0.7.4 ConfigSpace==0.4.10 scipy dataclasses optuna==2.3.0
```

Comment:
hmm, could we keep these unpinned for tests? I think we want to be as close to latest releases as possible!

Reply:
I removed the versions for everything but optuna. It looks like they pushed a v 2.4.0 really recently which is causing some changes in their api and breaks upstream. From what I can tell this was causing some of the tests to break fail. Not sure what your philosophy is on a flakiness issue like that is, but will adjust accordingly.

I did notice this: https://github.com/ray-project/ray/blob/master/python/requirements/requirements_tune.in , which lists 2.3 for optuna at the moment.